pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
```

Comment:
Ok, so how about "Report quota stats for all users, even those who aren't using any of their quota"?